### PR TITLE
capitalized y in youtube links

### PR DESCRIPTION
**Pull request by wbaccinelli**
Fixes https://github.com/PerfectFit-project/virtual-coach-issues/issues/440

In the NiceDay app, YouTube links work just if the 'y' is capital. The links in the resources have been adapted accordingly.

### CODE REVIEW

File: utils/activities.csv [hunk 1]
```diff
@@ -195,10 +195,10 @@ Maak een lijstje van dingen die je kunt doen in plaats van roken.✍️ Bepaal d
...
 
 Andere opties om stress en spanning te verminderen zijn:
-- Meditatie, mindfulness, of andere ademhalingstechnieken.🧘 Eenvoudige video’s hierover zijn te vinden op www.youtube.com. Zoek bijvoorbeeld op ‘5 minuten mindfulness meditatie’ of ‘begeleide beginners ademhalingsoefeningen’ of ‘mindfulness lichaamsscan’. Ook kan je een mindfulness of meditatie app downloaden (bijvoorbeeld de gratis app VGZ Mindfulness coach).
+- Meditatie, mindfulness, of andere ademhalingstechnieken.🧘 Eenvoudige video’s hierover zijn te vinden op www.Youtube.com. Zoek bijvoorbeeld op ‘5 minuten mindfulness meditatie’ of ‘begeleide beginners ademhalingsoefeningen’ of ‘mindfulness lichaamsscan’. Ook kan je een mindfulness of meditatie app downloaden (bijvoorbeeld de gratis app VGZ Mindfulness coach).
```
Comment (nelealbers) [inline utils/activities.csv:201]: I see that here the http-part is not there. This does not make a difference when it comes to the links working?

**Review by nelealbers — APPROVED** [review]
Looks good - have not tried the links myself though. So I was wondering if the capital Y works independently of whether the http part is there or not (just because I do not quite understand why the capital Y is needed).

File: utils/activities.csv (hunk 1 above)
Comment (wbaccinelli) [reply utils/activities.csv:201]: No, there is no difference, they work the same

**Comment by wbaccinelli** [comment]
> Looks good - have not tried the links myself though. So I was wondering if the capital Y works independently of whether the http part is there or not (just because I do not quite understand why the capital Y is needed).

I also don't understand why. This was a quickfix from NiceDay, but they are also looking into this issue.

**Pull request merged**